src/Rules/AlphaNew

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Rakit\Validation\Rules;
+
+use Rakit\Validation\Rule;
+
+class AlphaNew extends Rule
+{
+
+    protected $message = "The Place Of Birth only allows alphabet and numeric";
+
+    public function check($value)
+    {
+       
+
+        return preg_match('/^[\pL\pM 0-9 \,\.\-\/]+$/u', $value) > 0;
```

Comment:
I would say that idea behind this validator is to wide to be used. 
Also you got two spaces inside this regex